apps/nowcasting-app/components/charts/gsp-pv-remix-chart/use-get-gsp-data.ts

```diff
@@ -33,7 +26,22 @@ const useGetGspData = (gspId: number) => {
     refreshInterval: t5min
   });
 
-  const { data: gsp4HourData, error: pv4HourError } = useSWR<ForecastValue[]>(
+  //add new useSWR for gspChartData
+  const { data: gspForecastDataOneGSP, error: gspForecastDataOneGSPError } = useSWR<ForecastData>(
+    `${API_PREFIX}/solar/GB/gsp/forecast/${gspId}?forecast_horizon_minutes=0&historic=false&only_forecast_values=true`,
```

Comment:
I think the `forecast_horizon_minutes` has a default value of 0 and can't see any reason that would be changed, but doesn't matter too much

Reply:
I would remove forecast_horizon_minutes=0 if its not needed, without it, I think the you'll get the latest and it will be quicker

Reply:
or should this be `forecast_horizon_minutes=240`?

Reply:
I did think this, but the other day Postman told me off for not giving a "required" parameter `forecast_horizon_minutes` so I was wondering if something had changed in the API? (Though obviously not enough to look at it myself immediately ha)

Reply:
hmm, yea, have to check in the api - https://github.com/openclimatefix/uk-pv-national-gsp-api/blob/main/src/gsp.py#L108
looks optional

Reply:
Cool, yep in which case I can remove when I fix the conflicts on this branch 😊